packages/flutter/lib/src/material/text_field.dart

```diff
@@ -596,6 +599,9 @@ class TextField extends StatefulWidget {
   ///    which are more specialized input change notifications.
   final ValueChanged<String>? onChanged;
 
+  /// {@macro flutter.widgets.editableText.onContentCommitted}
```

Comment:
This macro doesn't exist (see my other comment where I explained how to fix this).